notebooks/TFoS.ipynb

```diff
@@ -232,9 +235,15 @@
    },
    "outputs": [],
    "source": [
-    "#verify test labels\n",
-    "test_labels_files = \"mnist/csv/test/labels\"\n",
-    "print(subprocess.check_output(shlex.split('hdfs dfs -ls {}'.format(test_labels_files))))"
+    "# Parse arguments for inference\n",
+    "args = parser.parse_args(['--mode', 'inference', '--epochs', '3',\n",
```

Comment:
defining epochs for inference feels weird, can we doublecheck we need all those parameters for inference?

Reply:
agreed and fixed